lib/js/jsre.nim

```diff
@@ -39,8 +28,27 @@ func compile*(self: RegExp; pattern: cstring; flags: cstring) {.importjs: "#.com
 func exec*(self: RegExp; pattern: cstring): seq[cstring] {.importjs: "#.exec(#)".}
   ## Executes a search for a match in its string parameter.
 
-func test*(self: RegExp; pattern: cstring): bool {.importjs: "#.test(#)".}
-  ## Tests for a match in its string parameter.
+func match*(self: RegExp; pattern: cstring): bool {.importjs: "#.test(#)".}
```

Comment:
> func newRegExp*(pattern: cstring; flags: cstring): RegExp {.importjs: "new RegExp(@)".}

`flags` is optional, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/RegExp